README.md

```diff
@@ -1,144 +1,97 @@
-This is a work in progress - don't expect it to be bug-free, secure or stable.
-
-The default configuration sets up a Bitcoin Core node and c-lightning. The user can enable spark-wallet in `configuration.nix` to make c-lightning accessible with a smartphone using spark-wallet.
-A simple webpage shows the lightning nodeid and links to nanopos letting the user receive donations.
-It also includes elements-daemon.
-Outbound peer-to-peer traffic is forced through Tor, and listening services are bound to onion addresses.
-
-A demo installation is running at [http://6tr4dg3f2oa7slotdjp4syvnzzcry2lqqlcvqkfxdavxo6jsuxwqpxad.onion](http://6tr4dg3f2oa7slotdjp4syvnzzcry2lqqlcvqkfxdavxo6jsuxwqpxad.onion).
-The following screen cast shows a fresh deployment of a nix-bitcoin node.
-
+<br/>
 <p align="center">
-  <a href="https://asciinema.org/a/223630/?speed=2&autoplay=1"><img src="https://asciinema.org/a/223630.png" height="500"></a>
+  <img
+    width="500"
+    src="https://raw.githubusercontent.com/nixbitcoin/nix-bitcoin/update-readme/docs/img/nix-bitcoin-logo-text.png"
+    alt="nix-bitcoin logo">
 </p>
 
+<br/>
+<p align="center">
+    <a href="https://github.com/fort-nix/nix-bitcoin/blob/master/LICENSE" target="_blank">
+        <img src="https://img.shields.io/github/license/fort-nix/nix-bitcoin" alt="GitHub license">
+    </a>
+    <a href="https://github.com/fort-nix/nix-bitcoin/releases/latest" target="_blank">
+        <img src="https://img.shields.io/github/v/release/fort-nix/nix-bitcoin" alt="GitHub tag (latest SemVer)">
+    </a>
+    <a href="https://cirrus-ci.com/github/fort-nix/nix-bitcoin" target="_blank">
+        <img src="https://api.cirrus-ci.com/github/fort-nix/nix-bitcoin.svg?branch=master" alt="CirrusCI actions status">
+    </a>
+    <a href="https://github.com/fort-nix/nix-bitcoin/commits/master" target="_blank">
+        <img src="https://img.shields.io/github/commit-activity/y/fort-nix/nix-bitcoin" alt="GitHub commit activity">
+    </a>
+    <a href="https://github.com/fort-nix/nix-bitcoin/graphs/contributors" target="_blank">
+        <img src="https://img.shields.io/github/contributors-anon/fort-nix/nix-bitcoin" alt="GitHub contributors">
+    </a>
+    <a href="https://github.com/fort-nix/nix-bitcoin/releases" target="_blank">
+        <img src="https://img.shields.io/github/downloads/fort-nix/nix-bitcoin/total" alt="GitHub downloads">
+    </a>
+</p>
+<br/>
 
+nix-bitcoin is a collection of Nix packages and NixOS modules for easily installing **full-featured Bitcoin nodes** with an emphasis on **security**.
 
-The goal is to make it easy to deploy a reasonably secure Bitcoin node with a usable wallet.
-It should allow managing bitcoin (the currency) effectively and providing public infrastructure.
-It should be a reproducible and extensible platform for applications building on Bitcoin.
-
-Examples
+Goals
```

Comment:
I'd suggest to replace the whole Goals section with something like the following:

Overview
---
A Bitcoin node verifies the Bitcoin protocol and provides ways of interacting with the Bitcoin network. nix-bitcoin nodes are used for a variety of purposes and can serve as personal or merchant wallets, second layer public infrastructure and as backends for Bitcoin applications. In all cases, the aim is to provide security and privacy by default. However, while nix-bitcoin is used in production today, it is still considered experimental.

A full installation of nix-bitcoin is usually deployed either on a dedicated (virtual) machine or runs in a container and is online 24/7. Alternatively, the Nix packages, NixOS modules and configurations can be used independently and combined freely.

nix-bitcoin is built on top of Nix and NixOS because that allows using abstractions powerful enough to keep it highly customizable and maintainable. Testament to this are nix-bitcoin's security features and its potent test framework. However, running nix-bitcoin does not require any previous experience with the nix ecosystem.